butlerx/wetty
# docker wf

- Bump vm2 from 3.9.3 to 3.9.5 (#362)
- format documentation (#363)
- Consistent WeTTY/wetty capitalization. (Upstream #360) (#364)
- issue #366: auto push docker images to dockerhub
- build latest docker image
- attempt to pusblish to github package
- Release 2.1.2
- change default entry point to yarn start
- issue #333: clear old terminal on start
- Check env version before spawning pty
- increase time for when tickets marked as stale
- Release 2.1.3
- fix release workflow
- issue #346: bundle configs in npm package
- issue #372: resize term on resize event
- Issue 365: allow host to be specified from http host headers
- Release 2.2.0
- Manually parse package.json to ensure version is always available
- Issue #368: fix allow i frame flag
- Release 2.2.1
- Add ability to set log level and move http logs to level http
- Release 2.3.0
- fix typo in publish workflow
- add prometheus metrics endpoint
- add metric for number of active connections
- time http requests
- xframe-disabled
- Bump vm2 from 3.9.5 to 3.9.7
- upgrade dependencies
- fix type errors
- update socket.io  version
- Release 2.4.0
- Rather then -> Rather than
- Fix download bug
- Bump minimist from 1.2.5 to 1.2.6
- Release 2.4.1
- Bump semver-regex from 3.1.3 to 3.1.4
- Bump jsdom from 15.2.1 to 16.5.0
- escape special chars in username: `- . \ @` (#406)
- fix lint warnings
- update docker build
- Bump cirrus-actions/rebase from 1.2 to 1.7 (#412)
- Bump pascalgn/npm-publish-action from 1.3.8 to 1.3.9 (#411)
- Bump actions/checkout from 2 to 3 (#410)
- Bump actions/stale from 3 to 5 (#409)
- Bump docker/login-action from 1 to 2 (#408)
- update lock file
- fix docker workflow if statement
